src/main/java/it/gov/pagopa/payhub/activities/activity/treasury/TreasuryIngestionActivity.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.payhub.activities.activity.treasury;
```

Comment:
Do you think it could be useful to have an other nested package for each treasury source?
```suggestion
package it.gov.pagopa.payhub.activities.activity.treasury.opi;
```